sdk/communication/Azure.Communication.Common/src/PhoneNumberIdentifier.cs

```diff
@@ -25,6 +28,33 @@ public override string RawId
         /// <summary>The phone number in E.164 format.</summary>
         public string PhoneNumber { get; }
 
+        /// <summary>True if the phone number is anonymous, e.g. when used to represent a hidden caller Id.</summary>
+        public bool IsAnonymous => RawId == $"{Phone}{Anonymous}";
+
+        /// <summary>The asserted Id is set on a phone number that is already in the same call to distinguish from other connections made through the same number.</summary>
+        public string AssertedId
+        {
+            get
+            {
+                if (_assertedId == "")
+                {
+                    return null;
+                }
+
+                var segments = RawId.Substring(Phone.Length).Split('_');
```

Comment:
This is strange, _assertedId is never used for reading

Reply:
it's returned on line 48th

Reply:
@paveldostalms I am not sure I understand the comment, could you please elaborate a bit? the PhoneNumber changes are anyway unrelated to OPS, so it shouldn't be a blocker for the alpha release

Reply:
I can see it, just it is always computed, set and then returned

Reply:
@DominikMe it seems we never return the _assertedId as it is, in case it already contains some value, but always recalculate it. Is this the intentional behavior? We will merge the PR as it is for now, in case some change will be needed here we can have a follow up